addons/account/views/account_reconcile_model_views.xml

```diff
@@ -108,40 +108,24 @@
                                             <group style="width:100% !important"
                                                    attrs="{'invisible': [('rule_type', '=', 'writeoff_button')]}">
                                                 <label for="match_amount"/>
-                                                <div>
-                                                    <field name="match_amount"
-                                                           style="width: 30% !important"
-                                                           class="oe_inline"/>
-                                                    <span class="o_form_label"
-                                                          style="width: 2% !important">   </span> <!-- Make a little space. -->
+                                                <div style="display: flex; gap: 5px;">
```

Comment:
class='d-flex gap-x' ? with x being a number between 1 and 5

Reply:
Haven't tried that, i will do it now, thx !